### ISSUE TITLE
Implement decorate=none rule

### ISSUE THREAD

**Issue by t-wissmann** [herbstluftwm/herbstluftwm#1316]
For shaped windows (e.g. Notification windows), it makes sense to hide the decoration. This should be configurable via a client's attribute (bool `decorated`) and a rule consequence to configure it automatically for many clients.

**Comment by allumik** [comment]
While this is implemented (hopefully), is there a way to simulate this with `hook` with `set attr` on currently focused windows?

Im looking for a way to disable borders and titles (im guessing these fall under *decoration*) for GTK apps and for browsers and such which could be specified with `class` or `title` conditions.

**Comment by t-wissmann** [comment]
Yes, most probably, I will try to make it configurable via an attribute.

**Comment by blarz** [comment]
Didn't #1374 implement this?

**Comment by t-wissmann** [comment]
There's only the attribute so far but no rule consequence yet.
